### DISCUSSION TITLE
How to add new vulnerabilities and bind them to corresponding components.

### DISCUSSION THREAD

**Discussion by icezhaoL** [DependencyTrack/dependency-track#1121] [General]
How to add new vulnerabilities and bind them to corresponding components.?
dependency-track 4.*  【Private Vulnerability Repository】feature  Will  be added ？
ths

**Comment by stevespringett** [comment]
Yes, private vulnerability repository was a preview (beta) feature in 3.x and was never completed. It will be completed in a future release.

**Reply by icezhaoL** [reply]
Look forward to it. Thank you very much
